pages/conduct.md

```diff
@@ -7,20 +7,20 @@ Like the technical community as a whole, the various jQuery Foundation teams and
 
 Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
 
-This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
+This isn't an exhaustive list of things that you can't do. Rather, take it in the spirit in which it's intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
 
-This code of conduct applies to all spaces managed by the jQuery Foundation and its projects. This includes IRC, the mailing lists, the issue trackers, jQuery Foundation events, and any other forums created by the project team which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+This code of conduct applies to all spaces managed by the jQuery Foundation and its projects. This includes IRC, the mailing lists, the issue trackers, jQuery Foundation events, and any other forums created by the project teams which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
 
-If you believe someone is violating the code of conduct, we ask that you report it by emailing conduct@jquery.org. For more details please see our [Reporting Guidelines](https://jquery.org/conduct/reporting/)
+If you believe someone is violating the code of conduct, we ask that you report it by emailing [conduct@jquery.org](mailto:conduct@jquery.org). For more details, please see our [Reporting Guidelines](/conduct/reporting/).
```

Comment:
I assume we omit the domain in other documents as well? One drawback is that links won't work when you read the Markdown file directly if we care about it.